TgeeT/AutoMart
# #166021858 fix specific car view page and links

#### What does this PR do?
Fix the view of a specific car
#### Description of Task to be completed?
Rearrange elements on the specific car view page to be more appealing
#### How should this be manually tested?
After cloning this repository, open the index.html file in your browser and click the name of any car to see specific details
#### Any background context you want to provide?
This is just a mare interface with no working functionalities
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166021858
#### Specific car details view
![image](https://user-images.githubusercontent.com/45405036/57966562-587c3d00-795c-11e9-841e-aa6aff00351e.png)
